### PR TITLE
Fix linting errors

### LINKED ISSUE #592: Lint convenience packages
### What

Fix all linting errors in the following packages:
- [ ] civic-babel-presets
- [ ] dev-server
- [ ] mock-wrapper
- [ ] webpack-common

This is a great opportunity to knock out an issue and familiarize yourself with CIVIC's convenience packages.

### Why

Improve code quality (#515)

### When

Not time sensitive, but as soon as possible

### PR DESCRIPTION
**Pull request by davidfloyd91**
Fixes #592

### CODE REVIEW

File: packages/dev-server/package.json [hunk 1]
```diff
@@ -17,10 +17,14 @@
...
+    "chalk": "^2.4.2",
+    "compression": "^1.7.4",
+    "express": "^4.17.1",
     "redux-logger": "^3.0.6",
+    "webpack": "^4.33.0",
     "webpack-dev-middleware": "^3.6.1",
     "webpack-hot-middleware": "^2.22.0"
-  },
-  "gitHead": "b50f7a7fca617964ade5a3ecb5225fee17b20fe4"
+  }
```
Comment (jaronheard) [inline packages/dev-server/package.json]:
```suggestion
  },
  "gitHead": "b50f7a7fca617964ade5a3ecb5225fee17b20fe4"
```

Comment (davidfloyd91) [reply packages/dev-server/package.json]: It looks like it's still there in line 20. Sort of weird how the diff is showing it, not sure why that's happening. Let me know if I'm missing something though

Comment (jaronheard) [reply packages/dev-server/package.json]: Ah, strange! Thanks for checking!

**Comment by davidfloyd91** [comment]
Thanks Jaron, happy to help. I think I searched open issues for a first good issue. So yeah totally just stumbled on it. Really cool project though

**Pull request merged**